### ISSUE TITLE
Summary component change link needs to be updated to allow a custom visually hidden span rather than piping in automatically

### ISSUE THREAD

**Issue by emilymaylawes** [ONSdigital/design-system#2769]
The [summary component](https://service-manual.ons.gov.uk/design-system/components/summary#hub) has 'Change' links on the right hand side that originally did not have unique names. We initially fixed this in https://github.com/ONSdigital/design-system/issues/2679 to add visually hidden text.

The solution we decided on was to pipe in the question and add "answer for {questionTitle}" (or similar). 

However, after investigating the usage of the summary component on the [hub and spoke](https://service-manual.ons.gov.uk/design-system/patterns/hub-and-spoke), it is noted that the grammar now does not make sense.

"View answers answer for People who live here"
"Start section answer for John Smith"

Whilst unique, not failing Label in Name, and has both the question and the link's label in its accessible name, it is not ideal and must be rectified.

It is recommended to create a new macro option, hiddenSpan (or named appropriately) that the user must individually provide.

For example,
"View answers {for section People who live here}"
"Start section {John Smith}"

Using visually hidden instead of the original `ariaLabel` is still recommended as it works when CSS is turned off, or if aria is not supported. It is also much more likely to not fail Label In Name as the visible label of the link is always included.

This may be a breaking change.

When we fix this, we must also update the content in the section Accessibility to reflect the new change.
